fuel_test/fuel_test.sh

```diff
@@ -2,6 +2,17 @@
 
 set -eux
 
+# If XCP is 1.9.0 (XS6.54) and branch is master, no need test
```

Comment:
Understood we have to skip XS6.5 on master as master now supports vxlan but 6.5 does not. So I've changed zuul to skip xs6.5. So we can drop this change now.